src/Eloquent/Builder.php

```diff
@@ -153,14 +153,6 @@ public function decrement($column, $amount = 1, array $extra = [])
         return parent::decrement($column, $amount, $extra);
     }
 
-    /**
-     * @inheritdoc
-     */
-    public function chunkById($count, callable $callback, $column = '_id', $alias = null)
-    {
-        return parent::chunkById($count, $callback, $column, $alias);
```

Comment:
Noted your response in https://github.com/GromNaN/laravel-mongodb/pull/25#discussion_r1300692739.

I missed that the meaningful change here was removing the `_id` parameter default value, since Laravel now consults the configured primary key (https://github.com/laravel/framework/pull/28065).